server/templates/staff/course/assignment/assignment.scores.html

```diff
@@ -81,6 +81,9 @@ <h3 class="box-title">Actions</h3>
                   <li> <a href="{{ url_for('.audit_scores', cid=current_course.id, aid=assignment.id) }}" type="button">
                         <i class="fa fa-search"></i> Find Submissions Without Scores
                   </a></li>
+                  <li> <a href="{{ url_for('.send_scores_job', cid=current_course.id, aid=assignment.id) }}" type="button">
```

Comment:
Nit: At this point - it's worth getting these into a button block instead of having them be nav-pills. 

Probably won't change it in this PR.